### PR TITLE
Update Bulgarian to include latest strings.

### PR DESCRIPTION
**Pull request by proudlytm**

### CODE REVIEW

**Comment by Akianonymus** [comment]
@proudlytm Nice work boi

**Comment by proudlytm** [comment]
@Akianonymus thanks

**Comment by FacuM** [comment]
@proudlytm Thanks m8, I'll accept it ASAP :3.

File: lang/bg_BG.php [hunk 1]
```diff
@@ -75,5 +82,18 @@
   // All
   'no_nick'                   => 'Анонимен',
   'time_seconds'              => ' секунди'
```
Comment (killer7Mod) [inline lang/bg_BG.php]: is missing the comma to separate the array

**Comment by FacuM** [comment]
Thanks for your help guys, merging!

**Pull request merged**